09-lambda/lab9-1-1/template.yml

```diff
@@ -0,0 +1,140 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: 'Creates lambda resources for Stelligent-U Topic 9: Lab 9-1-2'
+Resources: 
+  LambdaIamRole:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - lambda.amazonaws.com
+            Action:
+              - 'sts:AssumeRole'
+      Policies:
+        - PolicyName: AllowBasicLambda
+          PolicyDocument:
+            Version: "2012-10-17"
+            Statement:
+              - Effect: Allow
+                Action:
+                  - 'logs:CreateLogGroup'
+                Resource: "arn:aws:logs:us-east-1:324320755747:*"
+              - Effect: Allow
+                Action: 
+                  - logs:CreateLogStream
+                  - logs:PutLogEvents
+                Resource: arn:aws:logs:us-east-1:324320755747:log-group:/aws/lambda/bgar-lab-9-1-1-manual:*
```

Comment:
If this is outside your template you should create a param for it and ref that, other wise you should ref the resource